lib_eio_windows/fs.ml

```diff
@@ -89,6 +89,11 @@ end = struct
 
   let v ~label ~sandbox dir_path = { dir_path; sandbox; label; closed = false }
 
+  let chmod (_t : t) ~(follow : bool) ~(perm : int) (_path : string) : unit =
```

Comment:
Like with the other functions that do not have an implementation, we should actually still create a stub in `eio_windows_stubs.c` that fails. See, for example, how symlink does it. This reduces the burden for someone to come along and do a `chmod` implementation for Windows.